stdlib/LinearAlgebra/test/ldlt.jl

```diff
@@ -13,6 +13,20 @@ Random.seed!(123)
     lstring = sprint((t, s) -> show(t, "text/plain", s), F.L)
     dstring = sprint((t, s) -> show(t, "text/plain", s), F.D)
     @test ldltstring == "$(summary(F))\nL factor:\n$lstring\nD factor:\n$dstring"
-end
+# @testset "REPL printing of LDLT" begin
+#    S = SymTridiagonal(randn(5), randn(4))
+#    F = ldlt(S)
+#    ldltstring = sprint((t, s) -> show(t, "text/plain", s), F)
+#    lstring = sprint((t, s) -> show(t, "text/plain", s), F.L)
+#    dstring = sprint((t, s) -> show(t, "text/plain", s), F.D)
+#     @test ldltstring == "$(summary(F))\nL factor:\n$lstring\nD factor:\n$dstring"
+# end
```

Comment:
Sorry, I guess my code suggestion was misleading. Please don't add these lines, but rather comment out the ones above.